setup.py

```diff
@@ -68,6 +73,17 @@ def compilation_help():
 if not hasattr(sys, 'version_info') or sys.version_info < (2,7):
     compilation_help()
     raise SystemExit("Pygame requires Python version 2.7 or above.")
+if sys.version_info >= (3, 0) and sys.version_info < (3, 5):
+    compilation_help()
+    raise SystemExit("Pygame requires Python3 version 3.5 or above.")
+if IS_PYPY and sys.version_info < (3, 5):
+    raise SystemExit("Pygame requires PyPy compatible with Python 3.5 or above.")
```

Comment:
Last I looked, pypy (the 2.7 equivalent) worked. Am I misunderstanding this message?
I guess we should find the exact pypy version that pygame supports 'well enough'?

Reply:
I haven't yet tested on PyPy 2.7, this might be a bit too conservative.